package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "ast-types": "git+https://github.com/jlongster/ast-types.git",
     "babylon": "^6.15.0",
+    "esutils": "^2.0.2",
```

Comment:
Normally I'm very hesitant to add dependencies, but this looks like a neat library and isn't very large. Thanks!